persistence/upgrades/ravendb-3to4.md

```diff
@@ -37,6 +37,21 @@ NServiceBus now uses the asynchronous RavenDB API for all operations. If sharing
 Previously the API exposed an [`IDocumentSession`](https://ravendb.net/docs/search/latest/csharp?searchTerm=IDocumentSession), but now exposes [`IAsyncDocumentSession`](https://ravendb.net/docs/search/latest/csharp?searchTerm=IAsyncDocumentSession) instead, which contains the same operations but using a Task-based API.
 
 
+## Migrate saga data if using AllowStaleSagaReads
+
+Due to changes in how NServiceBus Version 6 handles saga correlation properties, solutions that previously used the `AllowStaleSagaReads()` option in RavenDB Persistence will not work properly and need to be migrated during upgrade.
+
+DANGER: Failure to migrate saga data when using the `AllowStaleSagaReads()` option will result in NServiceBus being unable to locate saga data, and then creating duplicate saga data erroneously.
+
+Normally, RavenDB persistence will use a _pointer document_ to efficiently load the saga data document by the correlation property in an atomic and consistent manner, without using RavenDB indexes that are (by design) not updated atomically with the document store.
```

Comment:
"use a _pointer document_ to efficiently load" sounds weird? perhaps something along the line of "to enable efficient loading.."?